### DISCUSSION TITLE
接口获取失败

### DISCUSSION THREAD

**Discussion by bzmshang** [jeessy2/ddns-go#995] [General]
2024/01/25 21:56:48 通过接口获取IPv4失败! 接口地址: https://myip4.ipip.net
2024/01/25 21:56:48 异常信息: Get "https://myip4.ipip.net": net/http: TLS handshake timeout

请问这个情况正常吗
![image](https://github.com/jeessy2/ddns-go/assets/111360926/d60d9934-fc15-4498-aba6-35b8470306c9)

**Comment by jeessy2** [comment]
正常，第二个接口起作用了

**Reply by bzmshang** [reply]
嗯嗯，感谢解答
